<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceData_Breaker</class>
 <widget class="QWidget" name="DeviceData_Breaker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: black;
}
QLabel
{
color:white;
background-color:rgb(240,0,0);
border:1px solid white;
}</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>20</y>
     <width>191</width>
     <height>348</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="LBL_SIV">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(102, 153, 255);</string>
      </property>
      <property name="text">
       <string>辅助(460V/230V)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="LBL_C1_CPR">
      <property name="text">
       <string>C1-CPR</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="LBL_C2_CPR">
      <property name="text">
       <string>C2-CPR</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="LBL_C3_CPR">
      <property name="text">
       <string>C3-CPR</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="LBL_DJ_CPR">
      <property name="text">
       <string>DJ-CPR</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="LBL_Q_REC">
      <property name="text">
       <string>Q-REC</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="LBL_CVT_BM1">
      <property name="text">
       <string>CVT-BM1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="LBL_CVT_BM2">
      <property name="text">
       <string>CVT-BM2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="LBL_CVT_MT1">
      <property name="text">
       <string>CVT-MT1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="LBL_CVT_MT2">
      <property name="text">
       <string>CVT-MT2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="LBL_CVT_SM1">
      <property name="text">
       <string>CVT-SM1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="LBL_CVT_SM2">
      <property name="text">
       <string>CVT-SM2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="LBL_DJ_PE1">
      <property name="text">
       <string>DJ-PE1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="LBL_DJ_PE2">
      <property name="text">
       <string>DJ-PE2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>455</x>
     <y>70</y>
     <width>191</width>
     <height>246</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="LBL_DJ_PE3">
      <property name="text">
       <string>DJ-PE3</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="LBL_DJ_PE4">
      <property name="text">
       <string>DJ-PE4</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="LBL_DJ_PH1">
      <property name="text">
       <string>DJ-PH1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="LBL_DJ_PH2">
      <property name="text">
       <string>DJ-PH2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="LBL_DJ_VT_SM1">
      <property name="text">
       <string>DJ-VT-SM1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="LBL_DJ_VT_SM2">
      <property name="text">
       <string>DJ-VT-SM2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="LBL_Q_TH_VT_BM1">
      <property name="text">
       <string>Q-TH-VT-BM1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="LBL_Q_TH_VT_BM2">
      <property name="text">
       <string>Q-TH-VT-BM2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="LBL_Q_TH_VT_MT1">
      <property name="text">
       <string>Q-TH-VT-MT1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="LBL_Q_TH_VT_MT2">
      <property name="text">
       <string>Q-TH-VT-MT2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="LBL_Train3Num">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>375</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777212</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font:10px;
border:black;
color:rgb(255,255,255);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>HXD2  8011C</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_Train2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>394</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="whatsThis">
    <string>1</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/TrainLogoUp-reverse.png);
background-color:black;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_Train3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>394</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="whatsThis">
    <string>2</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/TrainLogoUp.png);
background-color:black;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_Train4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>394</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="whatsThis">
    <string>3</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/TrainLogoUp-reverse.png);
background-color:black;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Train1Num">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>375</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777212</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font:10px;
border:black;
color:rgb(255,255,255);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>HXD2  8011A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Train4Num">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>375</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777212</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font:10px;
border:black;
color:rgb(255,255,255);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>HXD2  8011D</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_Train1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>394</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="whatsThis">
    <string>0</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/TrainLogoUp.png);
background-color:black;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Train2Num">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>375</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777212</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font:10px;
border:black;
color:rgb(255,255,255);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>HXD2  8011B</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
